### ISSUE TITLE
sqns firmware 43818

### ISSUE THREAD

**Issue by krismc** [pycom/pycom-micropython-sigfox#379]
My latest GPY arrived with 43818 firmware in the lte modem. Since this new GPY is more stable on lte than my other GPYs I would like to upgrade the 41065 modem firmware in them to 43818. Where can I get a copy of the 43818 sqns firmware please?

**Comment by catalinio** [comment]
Hi @krismc,

We're in the process of testing the upgrade packages.

**Comment by krismc** [comment]
tnx, any idea when an upgdiff_41065-to-43818.dup file might be available?

**Comment by abatardi** [comment]
5 months later no update?

**Comment by krismc** [comment]
Yeah it's a pain. I tried some NB-IoT41019 firmware in my latest gpy & now I'm stuck with it, can't roll it back to the CATM1-43818 that came with it.

**Comment by JCBuck** [comment]
Been another month? Also wondering where this might be...
